run_cumulus_task.py

```diff
@@ -26,5 +32,10 @@ def run_cumulus_task(task_function, cumulus_message, context):
     full_event = adapter.loadRemoteEvent(cumulus_message)
     nested_event = adapter.loadNestedEvent(full_event, vars(context))
     message_config = nested_event.get('messageConfig', {})
-    task_response = task_function(nested_event, context)
+
+    try:
+        task_response = task_function(nested_event, context)
+    except:
+        raise Exception("WorkflowError")
```

Comment:
This should be checking to see if the error being caught is a `WorkflowError`.

If it is then the original message (`cumulus_message`) should be returned with `payload: null` and `exception: error.name`.

If it is not then the error should be re-raised.

Reply:
This check should also be done if message_adapter_disabled is truthy.